lib/onelogin/ruby-saml/logoutresponse.rb

```diff
@@ -54,14 +65,38 @@ def in_response_to
         end
       end
 
+      # @return [String|nil] Gets the Destination attribute from the Logout Response if exists.
+      #
+      def destination
+        @destination ||= begin
+          node = REXML::XPath.first(document, "/p:LogoutResponse", { "p" => PROTOCOL })
+          node.nil? ? nil : node.attributes['Destination']
+        end
+      end
+
+      # @return [String] Gets the Issuer from the Logout Response.
+      #
       def issuer
         @issuer ||= begin
           node = REXML::XPath.first(document, "/p:LogoutResponse/a:Issuer", { "p" => PROTOCOL, "a" => ASSERTION })
-          node ||= REXML::XPath.first(document, "/p:LogoutResponse/a:Assertion/a:Issuer", { "p" => PROTOCOL, "a" => ASSERTION })
           node.nil? ? nil : node.text
         end
       end
 
+      # Checks if the Status has the "Success" code
+      # @param soft [Boolean] soft Enable or Disable the soft mode (In order to raise exceptions when the logout response is invalid or not)
+      # @return [Boolean] True if the StatusCode is Sucess
+      # @raise [ValidationError] if soft == false and validation fails
+      # 
+      def success?(soft = true)
+        unless status_code == "urn:oasis:names:tc:SAML:2.0:status:Success"
+          return soft ? false : validation_error("Bad status code. Expected <urn:oasis:names:tc:SAML:2.0:status:Success>, but was: <#@status_code> ")
```

Comment:
Please use proper interpolation here, the `#@status_code` syntax is asking for trouble, and will be incompatible with future versions of Ruby.